OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -70,6 +70,16 @@ function locate_java() {
 
     JAVA_HOME="$(dirname "$(dirname "$(readlink -f "${java}")")")"
     export JAVA_HOME
+
+    local libjvm
+    libjvm="$(find "${JAVA_HOME}" -name libjvm.so | head -1)"
```

Comment:
Since `-e` is used, a missing `libjvm.so` will cause the script to terminate unexpectedly. Which means line 77 will never be executed.

Changes need to be tested!

Reply:
Sorry,  that statement is not true in this case. 
Even if `-e` is used, the shell does not exit immediately if the command is part of the test in an if statement.

I tested these changes and works just fine.